inputFiles/singlePhaseFlow/thermalCompressible_2d_base.xml

```diff
@@ -11,7 +11,7 @@
       targetRegions="{ region }">
       <NonlinearSolverParameters
         newtonTol="1.0e-6"
-        newtonMaxIter="10"/>
+        newtonMaxIter="100"/>
```

Comment:
why? is this case taking 100 nonlinear iterations?

Reply:
20 would be sufficient

Reply:
This number is to ensure the convergence of the benchmark case

Reply:
I should converge in 20. if not, then something is wrong.

Reply:
I can confirm that it converges in 20.